src/Datadog.Trace.ClrProfiler.Native/environment_variables.h

```diff
@@ -110,6 +110,9 @@ namespace environment
     // Sets whether to enable the CallTarget instrumentation mode
     const WSTRING calltarget_enabled = WStr("DD_TRACE_CALLTARGET_ENABLED");
 
+    // Custom internal tracer profiler path
+    const WSTRING internal_trace_profiler_path = WStr("DD_INTERNAL_TRACE_NATIVE_ENGINE_PATH");
```

Comment:
Should this be `DD_TRACE_INTERNAL_` as opposed to `DD_INTERNAL_TRACE_`?

Reply:
discussion about that naming here: https://github.com/DataDog/dd-trace-dotnet/pull/1577#discussion_r683797626

Reply:
https://github.com/DataDog/dd-trace-dotnet/pull/1577#discussion_r682653024

Reply:
For consistency across languages, I think this deserves at least checking with the other teams (and maybe updating the env vars RFC).

Reply:
Well, please note this environment variable cannot be overwrote by anyone, this is because the env var is set when the native loader cor profiler is initialized (app process) and read by the tracer cor profiler in the same process. So it's just a way to pass data from the loader to the target profiler (1 dll to another dll) and if someone tries to set a value this will be overwritten when the loader initialises.  

Also, this is only valid on a .NET scenarios where we have this CorProfiler path.

Reply:
Sounds like we could rename it later if we wanted to without breaking changes, not anything public. Never mind!